src/client/CMakeLists.txt

```diff
@@ -62,19 +62,6 @@ elseif(PYPRT_LINUX)
 	# GCC 8 needs explicit linking of C++17 std::filesystem lib
 	target_link_libraries(${CLIENT_TARGET} PRIVATE stdc++fs)
```

Comment:
Late comment regarding PRT 2.4 upgrade and related upgrade to GCC 9 on Linux -> linking to `stdc++fs` should not be necessary anymore according to GCC 9 release notes: https://gcc.gnu.org/gcc-9/changes.html

Reply:
Created PR #117